business/workloads.go

```diff
@@ -80,7 +82,13 @@ func isWorkloadIncluded(workload string) bool {
 }
 
 // GetWorkloadList is the API handler to fetch the list of workloads in a given namespace.
-func (in *WorkloadService) GetWorkloadList(criteria WorkloadCriteria) (models.WorkloadList, error) {
+func (in *WorkloadService) GetWorkloadList(ctx context.Context, criteria WorkloadCriteria) (models.WorkloadList, error) {
```

Comment:
This is another criteria to pass into the span.
Not a blocker, I leave a comment here to not forget but I'd recommend adding it in a future iteration.